src/main/drivers/rx/rx_spi.c

```diff
@@ -37,39 +37,34 @@
 #include "drivers/rcc.h"
 #include "drivers/system.h"
 
+#include "pg/pg.h"
+#include "pg/rx_spi.h"
+
 #include "rx_spi.h"
 
-#define DISABLE_RX()    {IOHi(DEFIO_IO(RX_NSS_PIN));}
-#define ENABLE_RX()     {IOLo(DEFIO_IO(RX_NSS_PIN));}
+#define DISABLE_RX()    {IOHi(busdev->busdev_u.spi.csnPin);}
+#define ENABLE_RX()     {IOLo(busdev->busdev_u.spi.csnPin);}
+
+static busDevice_t rxSpiDevice;
+static busDevice_t *busdev = &rxSpiDevice;
 
 void rxSpiDeviceInit(void)
 {
-    static bool hardwareInitialised = false;
-
-    if (hardwareInitialised) {
```

Comment:
Why was this removed, and do we know that this is safe to be done?

Reply:
I will check the possibility of multiple calls.